notebooks/02_models/notebook.pluto.jl

```diff
@@ -1,18 +1,22 @@
 ### A Pluto.jl notebook ###
-# v0.17.1
+# v0.17.4
 
 using Markdown
 using InteractiveUtils
 
 # ╔═╡ d09256dd-6c0d-4e28-9e54-3f7b3ca87ecb
 begin
+  	using CSV
+  	using DataFrames
+  	import Distributions
+	using ScientificTypes
 	using MLJ
+	import MLJModelInterface  # to get vers. 1.3.5
+	using MLJBase 			# to get vers. 0.19.2
```

Comment:
That's weird. I don't really understand why you need these extra imports, or how you are running into that issue. If I create a fresh  environment with the old package list I get MLJModelInterface 1.3.5 and MLJBase 0.19.2 (see below). Indeed MLJ 0.17 can only load MLJBase 0.19.2 because 0.19.0 and 0.19.1 have been pulled from the registry. And the problematic MLJModelInterface 1.3.4 has also been yanked. 

Perhaps you started before the General.jl updates became live??

```julia
(jl_luJE7O) pkg> st
      Status `/private/var/folders/4n/gvbmlhdc8xj973001s6vdyw00000gq/T/jl_luJE7O/Project.toml`
  [336ed68f] CSV v0.9.11
  [a93c6f00] DataFrames v1.3.1
  [31c24e10] Distributions v0.25.37
  [add582a8] MLJ v0.17.0
  [094fc8d1] MLJFlux v0.2.6
  [91a5bcdd] Plots v1.25.4
  [7f904dfe] PlutoUI v0.7.28
  [321657f4] ScientificTypes v3.0.0
  [bd369af6] Tables v1.6.1
  [856ac37a] UrlDownload v1.0.0

(jl_luJE7O) pkg> st -m
      Status `/private/var/folders/4n/gvbmlhdc8xj973001s6vdyw00000gq/T/jl_luJE7O/Manifest.toml`
  [da404889] ARFFFiles v1.4.1
  [621f4979] AbstractFFTs v1.0.1
  [6e696c72] AbstractPlutoDingetjes v1.1.4
  [1520ce14] AbstractTrees v0.3.4
  [79e6a3ab] Adapt v3.3.3
  [4fba245c] ArrayInterface v3.2.2
  [ab4f0b2a] BFloat16s v0.2.0
  [fbb218c0] BSON v0.3.4
  [fa961155] CEnum v0.4.1
  [336ed68f] CSV v0.9.11
  [052768ef] CUDA v3.6.2
  [324d7699] CategoricalArrays v0.10.2
  [af321ab8] CategoricalDistributions v0.1.4
  [082447d4] ChainRules v1.17.0
  [d360d2e6] ChainRulesCore v1.11.4
  [9e997f8a] ChangesOfVariables v0.1.2
  [944b1d66] CodecZlib v0.7.0
  [35d6a980] ColorSchemes v3.15.0
  [3da002f7] ColorTypes v0.11.0
  [5ae59095] Colors v0.12.8
  [bbf7d656] CommonSubexpressions v0.3.0
  [34da2185] Compat v3.41.0
  [ed09eef8] ComputationalResources v0.3.2
  [d38c429a] Contour v0.5.7
  [a8cc5b0e] Crayons v4.1.0
  [9a962f9c] DataAPI v1.9.0
  [a93c6f00] DataFrames v1.3.1
  [864edb3b] DataStructures v0.18.11
  [e2d170a0] DataValueInterfaces v1.0.0
  [b429d917] DensityInterface v0.4.0
  [163ba53b] DiffResults v1.0.3
  [b552c78f] DiffRules v1.9.0
  [b4f34e82] Distances v0.10.7
  [31c24e10] Distributions v0.25.37
  [ffbed154] DocStringExtensions v0.8.6
  [792122b4] EarlyStopping v0.3.0
  [e2ba6199] ExprTools v0.1.6
  [c87230d0] FFMPEG v0.4.1
  [48062228] FilePathsBase v0.9.17
  [1a297f60] FillArrays v0.12.7
  [53c48c17] FixedPointNumbers v0.8.4
  [587475ba] Flux v0.12.8
  [59287772] Formatting v0.4.2
  [f6369f11] ForwardDiff v0.10.24
  [d9f16b24] Functors v0.2.7
  [0c68f7d7] GPUArrays v8.1.3
  [61eb1bfa] GPUCompiler v0.13.10
  [28b8d3ca] GR v0.63.0
  [5c1252a2] GeometryBasics v0.4.1
  [42e2da0e] Grisu v1.0.2
  [cd3eb016] HTTP v0.9.17
  [47d2ed2b] Hyperscript v0.0.4
  [ac1192a8] HypertextLiteral v0.9.3
  [b5f81e59] IOCapture v0.2.2
  [7869d1d1] IRTools v0.4.4
  [615f187c] IfElse v0.1.1
  [83e8ac13] IniFile v0.5.0
  [842dd82b] InlineStrings v1.1.1
  [3587e190] InverseFunctions v0.1.2
  [41ab1584] InvertedIndices v1.1.0
  [92d709cd] IrrationalConstants v0.1.1
  [c8e1da08] IterTools v1.4.0
  [b3c1a2ee] IterationControl v0.5.2
  [82899510] IteratorInterfaceExtensions v1.0.0
  [692b3bcd] JLLWrappers v1.3.0
  [9da8a3cd] JLSO v2.6.0
  [682c06a0] JSON v0.21.2
  [e5e0dc1b] Juno v0.8.4
  [929cbde3] LLVM v4.7.0
  [b964fa9f] LaTeXStrings v1.3.0
  [23fbe1c1] Latexify v0.15.9
  [a5e1c1ea] LatinHypercubeSampling v1.8.0
  [7f8f8fb0] LearnBase v0.4.1
  [2ab3a3ac] LogExpFunctions v0.3.6
  [30fc2ffe] LossFunctions v0.7.2
  [add582a8] MLJ v0.17.0
  [a7f614a8] MLJBase v0.19.2      <------------------------------------------
  [50ed68f4] MLJEnsembles v0.2.0
  [094fc8d1] MLJFlux v0.2.6
  [614be32b] MLJIteration v0.4.1
  [e80e1ace] MLJModelInterface v1.3.5      <------------------------------
  [d491faf4] MLJModels v0.15.1
  [17bed46d] MLJSerialization v1.1.3
  [03970b2e] MLJTuning v0.6.16
  [1914dd2f] MacroTools v0.5.9
  [739be429] MbedTLS v1.0.3
  [442fdcdd] Measures v0.3.1
  [e89f7d12] Media v0.5.0
  [f28f55f0] Memento v1.3.0
  [e1d29d7a] Missings v1.0.2
  [872c559c] NNlib v0.7.31
  [a00861dc] NNlibCUDA v0.1.11
  [77ba4419] NaNMath v0.3.6
  [8b6db2d4] OpenML v0.2.0
  [bac558e1] OrderedCollections v1.4.1
  [90014a1f] PDMats v0.11.5
  [d96e819e] Parameters v0.12.3
  [69de0a69] Parsers v2.1.3
  [ccf2f8ad] PlotThemes v2.0.1
  [995b91a9] PlotUtils v1.1.2
  [91a5bcdd] Plots v1.25.4
  [7f904dfe] PlutoUI v0.7.28
  [2dfb63ee] PooledArrays v1.4.0
  [21216c6a] Preferences v1.2.3
  [54e16d92] PrettyPrinting v0.3.2
  [08abe8d2] PrettyTables v1.3.1
  [92933f4c] ProgressMeter v1.7.1
  [1fd47b50] QuadGK v2.4.2
  [74087812] Random123 v1.4.2
  [e6cf234a] RandomNumbers v1.5.3
  [c1ae055f] RealDot v0.1.0
  [3cdcf5f2] RecipesBase v1.2.1
  [01d81517] RecipesPipeline v0.4.1
  [189a3867] Reexport v1.2.2
  [ae029012] Requires v1.2.0
  [79098fc4] Rmath v0.7.0
  [321657f4] ScientificTypes v3.0.0
  [30f210dd] ScientificTypesBase v3.0.0
  [6c6a2e73] Scratch v1.1.0
  [91c51154] SentinelArrays v1.3.10
  [992d4aef] Showoff v1.0.3
  [a2af1166] SortingAlgorithms v1.0.1
  [276daf66] SpecialFunctions v2.0.0
  [860ef19b] StableRNGs v1.0.0
  [aedffcd0] Static v0.4.1
  [90137ffa] StaticArrays v1.3.0
  [64bff920] StatisticalTraits v3.0.0
  [82ae8749] StatsAPI v1.2.0
  [2913bbd2] StatsBase v0.33.14
  [4c63d2b9] StatsFuns v0.9.14
  [09ab397b] StructArrays v0.6.3
  [3783bdb8] TableTraits v1.0.1
  [bd369af6] Tables v1.6.1
  [a759f4b9] TimerOutputs v0.5.13
  [3bb67fe8] TranscodingStreams v0.9.6
  [5c2747f8] URIs v1.3.0
  [3a884ed6] UnPack v1.0.2
  [1cfade01] UnicodeFun v0.4.1
  [b8865327] UnicodePlots v2.5.1
  [41fe7b60] Unzip v0.1.2
  [856ac37a] UrlDownload v1.0.0
  [ea10d353] WeakRefStrings v1.4.1
  [a5390f91] ZipFile v0.9.4
  [e88e6eb3] Zygote v0.6.33
  [700de1a5] ZygoteRules v0.2.2
  [6e34b625] Bzip2_jll v1.0.8+0
  [83423d85] Cairo_jll v1.16.1+1
  [5ae413db] EarCut_jll v2.2.3+0
  [2e619515] Expat_jll v2.2.10+0
  [b22a6f82] FFMPEG_jll v4.4.0+0
  [a3f928ae] Fontconfig_jll v2.13.93+0
  [d7e528f0] FreeType2_jll v2.10.4+0
  [559328eb] FriBidi_jll v1.0.10+0
  [0656b61e] GLFW_jll v3.3.5+1
  [d2c73de3] GR_jll v0.63.0+0
  [78b55507] Gettext_jll v0.21.0+0
  [7746bdde] Glib_jll v2.68.3+2
  [3b182d85] Graphite2_jll v1.3.14+0
  [2e76f6c2] HarfBuzz_jll v2.8.1+1
  [aacddb02] JpegTurbo_jll v2.1.0+0
  [c1c5ebd0] LAME_jll v3.100.1+0
  [dad2f222] LLVMExtra_jll v0.0.13+0
  [dd4b983a] LZO_jll v2.10.1+0
  [e9f186c6] Libffi_jll v3.2.2+1
  [d4300ac3] Libgcrypt_jll v1.8.7+0
  [7e76a0d4] Libglvnd_jll v1.3.0+3
  [7add5ba3] Libgpg_error_jll v1.42.0+0
  [94ce4f54] Libiconv_jll v1.16.1+1
  [4b2f31a3] Libmount_jll v2.35.0+0
  [89763e89] Libtiff_jll v4.3.0+0
  [38a345b3] Libuuid_jll v2.36.0+0
  [e7412a2a] Ogg_jll v1.3.5+1
  [458c3c95] OpenSSL_jll v1.1.10+0
  [efe28fd5] OpenSpecFun_jll v0.5.5+0
  [91d4177d] Opus_jll v1.3.2+0
  [2f80f16e] PCRE_jll v8.44.0+0
  [30392449] Pixman_jll v0.40.1+0
  [ea2cea3b] Qt5Base_jll v5.15.3+0
  [f50d1b31] Rmath_jll v0.3.0+0
  [a2964d1f] Wayland_jll v1.19.0+0
  [2381bf8a] Wayland_protocols_jll v1.23.0+0
  [02c8fc9c] XML2_jll v2.9.12+0
  [aed1982a] XSLT_jll v1.1.34+0
  [4f6342f7] Xorg_libX11_jll v1.6.9+4
  [0c0b7dd1] Xorg_libXau_jll v1.0.9+4
  [935fb764] Xorg_libXcursor_jll v1.2.0+4
  [a3789734] Xorg_libXdmcp_jll v1.1.3+4
  [1082639a] Xorg_libXext_jll v1.3.4+4
  [d091e8ba] Xorg_libXfixes_jll v5.0.3+4
  [a51aa0fd] Xorg_libXi_jll v1.7.10+4
  [d1454406] Xorg_libXinerama_jll v1.1.4+4
  [ec84b674] Xorg_libXrandr_jll v1.5.2+4
  [ea2f1a96] Xorg_libXrender_jll v0.9.10+4
  [14d82f49] Xorg_libpthread_stubs_jll v0.1.0+3
  [c7cfdc94] Xorg_libxcb_jll v1.13.0+3
  [cc61e674] Xorg_libxkbfile_jll v1.1.0+4
  [12413925] Xorg_xcb_util_image_jll v0.4.0+1
  [2def613f] Xorg_xcb_util_jll v0.4.0+1
  [975044d2] Xorg_xcb_util_keysyms_jll v0.4.0+1
  [0d47668e] Xorg_xcb_util_renderutil_jll v0.3.9+1
  [c22f9ab0] Xorg_xcb_util_wm_jll v0.4.1+1
  [35661453] Xorg_xkbcomp_jll v1.4.2+4
  [33bec58e] Xorg_xkeyboard_config_jll v2.27.0+4
  [c5fb5394] Xorg_xtrans_jll v1.4.0+3
  [3161d3a3] Zstd_jll v1.5.0+0
  [0ac62f75] libass_jll v0.15.1+0
  [f638f0a6] libfdk_aac_jll v2.0.2+0
  [b53b4c65] libpng_jll v1.6.38+0
  [f27f6e37] libvorbis_jll v1.3.7+0
  [1270edf5] x264_jll v2021.5.5+0
  [dfaa095f] x265_jll v3.5.0+0
  [d8fb68d0] xkbcommon_jll v0.9.1+5
  [0dad84c5] ArgTools
  [56f22d72] Artifacts
  [2a0f44e3] Base64
  [ade2ca70] Dates
  [8bb1440f] DelimitedFiles
  [8ba89e20] Distributed
  [f43a241f] Downloads
  [9fa8497b] Future
  [b77e0a4c] InteractiveUtils
  [4af54fe1] LazyArtifacts
  [b27032c2] LibCURL
  [76f85450] LibGit2
  [8f399da3] Libdl
  [37e2e46d] LinearAlgebra
  [56ddb016] Logging
  [d6f4376e] Markdown
  [a63ad114] Mmap
  [ca575930] NetworkOptions
  [44cfe95a] Pkg
  [de0858da] Printf
  [9abbd945] Profile
  [3fa0cd96] REPL
  [9a3f8284] Random
  [ea8e919c] SHA
  [9e88b42a] Serialization
  [1a1011a3] SharedArrays
  [6462fe0b] Sockets
  [2f01184e] SparseArrays
  [10745b16] Statistics
  [4607b0f0] SuiteSparse
  [fa267f1f] TOML
  [a4e569a6] Tar
  [8dfed614] Test
  [cf7118a7] UUIDs
  [4ec0a83e] Unicode
  [e66e0078] CompilerSupportLibraries_jll
  [deac9b47] LibCURL_jll
  [29816b5a] LibSSH2_jll
  [c8ffd9c3] MbedTLS_jll
  [14a3606d] MozillaCACerts_jll
  [4536629a] OpenBLAS_jll
  [05823500] OpenLibm_jll
  [83775a58] Zlib_jll
  [8e850b90] libblastrampoline_jll
  [8e850ede] nghttp2_jll
  [3f19e933] p7zip_jll
```

Reply:
Thanks for the hint! You are right, these two imports are not needed. They stem from the phase where I tried to fetch these packages, but they weren't available yet.